helper/schema/field_reader.go

```diff
@@ -171,7 +171,7 @@ func addrToSchema(addr []string, schemaMap map[string]*Schema) []*Schema {
 // "foo.#" for a list "foo" and that the indexes are "foo.0", "foo.1", etc.
 // after that point.
 func readListField(
-	r FieldReader, addr []string, schema *Schema) (FieldReadResult, error) {
+	r FieldReader, addr []string) (FieldReadResult, error) {
```

Comment:
Mmm, why is this function signature formatted like this?

Reply:
We're not joking when we say a lot of this code is _old_ and generally frozen. Say hello to 2014! https://github.com/hashicorp/terraform/commit/944797301522b5784192fc2b05ffb4ca4695df03

Reply:
We can decide if it is worth going through all the old function signatures to fix them up because this trips me up for a second every time I see it too.